README.md

```diff
@@ -44,6 +44,38 @@ In order to configure some aspects of layout, like the header, edit `config/tech
 
 If you move pages around and URLs change, make sure you set up redirects from the old URLs to the new URLs.
```

Comment:
I'm wondering about the phrase "move pages around" here. It's nice and colloquial, but I wonder if it's a little too much that way? Maybe something like "If you change the location of pages, and their URLs change.." or something?

Reply:
@charleemg we should scope reviews to proposed changes within the PR,
if you're spotting things that are outside of the green bands in github, those are best addressed as issues, or backlog tickets.

But we should be careful not to review things here that are outside of the changes proposed by @pauldougan

Reply:
again, going to resolve this, feel free to reach out on slack if you have concerns about these comments?